backend/api.test/Mocks/HttpContextAccessorMock.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.AspNetCore.Http;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Api.Test.Mocks
+{
+    public class MockHttpContextAccessor : IHttpContextAccessor
+    {
+        public HttpContext? HttpContext
+        {
+            get
+            {
+                var context = new DefaultHttpContext();
+                var tokenHandler = new JwtSecurityTokenHandler();
+                var claims = new List<Claim>
+                {
+                    new(ClaimTypes.Role, "Role.Admin"),
+                    new(ClaimTypes.Role, "Role.User"),
+                    new(ClaimTypes.Role, "Role.User.JSV")
```

Comment:
Use Huldra for the test?

Reply:
This is not really possible unless we add huldra missions in echo as we use the installation code when fetching tag positions when running missions

Reply:
![image](https://github.com/equinor/flotilla/assets/20266900/21654c1d-907d-4476-96ef-0c5b6e2fca18)